Description: remove functions not present in Debian
Author: Martin <debacle@debian.org>
Origin: vendor
Last-Update: 2021-01-07
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/gulpfile.js
+++ b/gulpfile.js
@@ -1,8 +1,7 @@
 var gulp = require('gulp');
 var concat = require('gulp-concat');
-var uglify = require('gulp-uglify');
 var babel = require('gulp-babel');
-var filesExist = require('files-exist');
+var filesExist = function(fileArray, options) { return fileArray; };
 var maps = require('gulp-sourcemaps');
 
 var files = [
@@ -46,7 +45,6 @@
         ]
     }))
     .pipe(concat('jquery.flot.js'))
-    .pipe(uglify())
     .pipe(maps.write('./'))
     .pipe(gulp.dest('dist/es5'));
 }
